tf2_sensor_msgs/tf2_sensor_msgs/__init__.py

```diff
@@ -26,4 +26,9 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from tf2_sensor_msgs import *  # noqa(E401)
+from .tf2_sensor_msgs import do_transform_cloud, transform_points
```

Comment:
Just a minor comment:

Since the original PR also split these over 2 lines, I would prefer splitting it here too:

```suggestion
from .tf2_sensor_msgs import do_transform_cloud
from .tf2_sensor_msgs import transform_points
```

While our [style guide](https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html#id4) refers to [PEP 8](https://peps.python.org/pep-0008/), which [does allow multiple imports on the same line](https://peps.python.org/pep-0008/#imports), as you did here, we often just have 1 import per line.